platform-mbean/src/main/java/org/jboss/as/platform/mbean/RuntimeMXBeanAttributeHandler.java

```diff
@@ -116,7 +116,11 @@ static void storeResult(final String name, final ModelNode store) {
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH_SUPPORTED.equals(name)) {
             store.set(ManagementFactory.getRuntimeMXBean().isBootClassPathSupported());
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH.equals(name)) {
-            store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            if (!ManagementFactory.getRuntimeMXBean().isBootClassPathSupported()) {
+                store.set(""); // JDK9 and above does not have boot class path
+            } else {
+                store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            }
```

Comment:
this was my "hack" at the time to properly fix it https://github.com/ctomc/wildfly-core/commit/d5f31409304254c94a58f1d10c0f02852719f425 but this looks like problem in JDK as we don't get UnsupportedOperationException itself but only has that in message.